google/oauth2/service_account.py

```diff
@@ -302,6 +314,13 @@ def with_scopes(self, scopes, default_scopes=None):
         cred._default_scopes = default_scopes
         return cred
 
+    def lookup_trust_boundary(self, request):
+        url = _TRUST_BOUNDARY_LOOKUP_ENDPOINT.format(
+            self.universe_domain, self.service_account_email
+        )
+        headers = {"Authorization": "Basic " + self.token}
```

Comment:
I'm not familiar with this endpoint, but should Basic be Bearer?

Reply:
This endpoint is not done yet. The current design is that would be "Basic"